src/engine/outfit.ts

```diff
@@ -182,10 +293,30 @@ export function equipDefaults(outfit: Outfit, noFightingFamiliars: boolean): voi
     }
   }
 
+  // If I figure out how to add to combat macro:
+
+  /* if(modifier.includes("item"))
+    if(get("everfullDartPerks").includes("Butt awareness"))
+      // eslint-disable-next-line libram/verify-constants
+      outfit.addBonus($item`Everfull Dart Holster`, 30) */
+
   if (outfit.skipDefaults) return;
 
   if (modifier.includes("-combat")) outfit.equip($familiar`Disgeist`); // low priority
-  if (!noFightingFamiliars) outfit.equip($familiar`Jill-of-All-Trades`);
+
+  if (args.minor.profitFamiliar) {
```

Comment:
This doesn't need to be here both on line 307 and 313. Probably you only need the version on 313. (Or you can go through the profit familiars and see which ones attack. The main usage here is that we don't want to accidentally kill gremlins during the L12 quest when we are trying to stasis).